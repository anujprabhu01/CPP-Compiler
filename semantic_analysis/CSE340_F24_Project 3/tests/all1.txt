\rtf1\ansi\ansicpg1252\cocoartf2761
\cocoatextscaling0\cocoaplatform0{\fonttbl\f0\fnil\fcharset0 HelveticaNeue;}
{\colortbl;\red255\green255\blue255;\red0\green0\blue0;}
{\*\expandedcolortbl;;\cssrgb\c0\c0\c0;}
\margl1440\margr1440\vieww11520\viewh8400\viewkind0
\deftab720
\pard\tx220\tx720\pardeftab720\li720\fi-720\partightenfactor0

\f0\fs32 \cf0 \expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 a, b, c, d, e, f;\
\{\
    input a;\
    d = 1;\
    e = 2;\
    SWITCH a \{\
        CASE 1: \{\
            input b;\
            FOR (c = 1; c < 5; c = c + 1;) \{\
                b = b * 2;\
                output b;\
            \}\
            a = 2;\
            SWITCH a \{\
                CASE 1: \{\
                    output a;\
                \}\
                DEFAULT: \{\
                    IF d > e \{\
                        d = d * 2;\
                        output d;\
                    \}\
                    IF d < e \{\
                        e = e * 2;\
                        output e;\
                        d = 5;\
                        e = 6;\
                    \}\
                \}\
            \}\
        \}\
    \}\
\
    WHILE d > 2 \{\
        WHILE e > 1 \{\
            f = d * e;\
            output f;\
            e = e - 1;\
        \}\
        d = d - 1;\
    \}\
\
    output a;\
    output b;\
    output c;\
    output d;\
    output e;\
    output f;\
\}\
1111