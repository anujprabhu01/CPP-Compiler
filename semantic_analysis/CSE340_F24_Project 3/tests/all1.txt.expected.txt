2 4 8 16 4 30 25 20 15 10 2 16 5 2 1 10